virtcontainers/container.go

```diff
@@ -1490,6 +1496,11 @@ func (c *Container) cgroupsDelete() error {
 
 // updates container cgroups in the host
 func (c *Container) cgroupsUpdate(resources specs.LinuxResources) error {
+
+	if c.state.CgroupPath == "" {
+		c.Logger().Warn("container has not cgroup path nothing to update")
+		return nil
```

Comment:
Should these be warnings? Seems info or debug maybe?
Suggested comment: “container does not have cgroup on host: nothing to update”?